openwisp_radius/templates/djangosaml2/auth_error.html

```diff
@@ -1,24 +1,14 @@
-{% comment %}
-This template is taken from djangosaml2 with sole purpose of
-updating name space of the URL identifier.
-{% endcomment %}
+{% extends "account/base.html" %}
+{% load i18n %}
 
-<!DOCTYPE html
-    PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+{% block content %}
+    <h1> {% blocktrans %} Authorization error {% endblocktrans %} </h1>
```

Comment:
you should be able to use just `translate` or `trans` when it's just one line.

Can you please remove the shitespace between the HTML tags and the words please?

Eg:

`<h1>{% trans 'Word' %}</h1>`

Applies to other occurrencets as well.